src/parser/warrior/fury/modules/talents/AngerManagement.js

```diff
@@ -18,29 +22,35 @@ class AngerManagement extends Analyzer {
     spellUsable: SpellUsable,
   };
 
+  cooldownsAffected = [
```

Comment:
Looking at the talent I understand why this is an array, but for Fury it only affects 1 ability - Recklessness. Rewriting it to only take into account this fact (and **not** be generalized for array of spells) would simplify the module.